dwitter/static/js/scrolling.js

```diff
@@ -43,6 +43,11 @@ window.onload = function() {
               $(this).off("click.a keyup.a").select();
             });
           });
+
+          moment.locale(navigator.userLanguage || navigator.language || 'en-US');
+          var timestamp = $(div).find('.dweet-timestamp');
+          var postedDate = moment.utc(timestamp.text());
+          timestamp.text(postedDate.local().format('lll'));
```

Comment:
I totally agree, it's getting cumbersome and error prone to deal with. Some bigger refactoring, especially of most of the js code is needed 👍 I'd like to do that refactor myself; but it would most likely have to wait a week or two at least.